backend/src/akvo/lumen/import.clj

```diff
@@ -37,14 +37,16 @@
                                   :table-name table-name
                                   :imported-table-name imported-table-name
                                   :version 1
-                                  :columns (mapv (fn [{:keys [title id type key] :as columns}]
-                                                   (cond-> {:type (name type)
-                                                            :title (string/trim title)
-                                                            :columnName (name id)
-                                                            :sort nil
-                                                            :direction nil
-                                                            :hidden false}
-                                                     (contains? columns :key) (assoc :key (boolean key))))
+                                  :columns (mapv (fn [{:keys [title id type key caddisflyResourceUuid]}]
+                                                   (cond->
+                                                       {:type (name type)
+                                                        :title (string/trim title)
+                                                        :columnName (name id)
+                                                        :sort nil
+                                                        :direction nil
+                                                        :hidden false}
+                                                     caddisflyResourceUuid (assoc :caddisflyResourceUuid caddisflyResourceUuid)
```

Comment:
this change doesn't affect to older datasets updates so doesn't imply adaptations